app/code/Magento/Customer/view/frontend/layout/customer_account.xml

```diff
@@ -24,31 +24,31 @@
                         <arguments>
                             <argument name="label" xsi:type="string" translate="true">My Account</argument>
                             <argument name="path" xsi:type="string">customer/account</argument>
-                            <argument name="sortOrder" xsi:type="number">250</argument>
+                            <argument name="sortOrder" xsi:type="number">0</argument>
```

Comment:
I'm really wondering, how will we add another item before `My Account` as far as this one has `0`?

Reply:
If you used negative sortOrder the link will go first before 0. In this case new link will render before My Account link

Reply:
😄 In my opinion this isn't something correctly and obviously to have negative values for positions.
I'd suggest using a higher number instead of 0.